fanuc_lrmate200id_gazebo/urdf/lrmate200id.gazebo

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <!-- controllers -->
+
+  <gazebo>
+    <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
+      <robotNamespace>/</robotNamespace>
+      <robotSimType>gazebo_ros_control/DefaultRobotHWSim</robotSimType>
+    </plugin>
+  </gazebo>
+
+  <!-- fix the base to the Gazebo world -->
+
+  <link name="world" />
+  <joint name="fixed" type="fixed">
+    <origin xyz="0 0 0" rpy="0 0 0"/>
+    <parent link="world"/>
+    <child link="base_link"/>
+  </joint>
```

Comment:
What was your rationale for placing this *inside* this file? If users don't want to directly link the `fanuc_lrmate200id` model to the world, but to another entity in the simulated world, they would not be able to re-use this `fanuc_lrmate200id.gazebo` file.

That is one of the reasons we chose to keep that outside of the base `irb120_macro.xacro` at the time, and place it in the top-level `irb120.xacro` (those cannot be included in another xacro anyway, so placing the `world-base_link` joint there is not a problem).

Not saying this is wrong, I'm just curious to hear whether you had a reason for doing it this way.